testing/MLDB-162-min-max-function.py

```diff
@@ -118,4 +118,111 @@
+assert result == expected
+
+result = mldb.query('SELECT clamp({null, 11}, 3, 7) as v')
+expected = [["_rowName","v.null","v.11"],["result",None,7]]
+assert result == expected
+
+result = mldb.query('SELECT clamp(NaN, 3, 7) as v')
+expected = [["_rowName","v"],["result",3]]
+assert result == expected 
+
+result = mldb.query('SELECT clamp(-NaN, 3, 7) as v')
+expected = [["_rowName","v"],["result",3]]
+assert result == expected 
+
+result = mldb.query('SELECT clamp(inf, 3, 7) as v')
+expected = [["_rowName","v"],["result",7]]
+assert result == expected 
+
+result = mldb.query('SELECT clamp(-inf, 3, 7) as v')
+expected = [["_rowName","v"],["result",3]]
+assert result == expected 
+
+result = mldb.query('SELECT clamp(2, null, 3) as v')
+expected = [["_rowName","v"],["result",2]]
+assert result == expected 
+
+result = mldb.query('SELECT clamp(2, null, 1) as v')
+expected = [["_rowName","v"],["result",1]]
+assert result == expected
+
+result = mldb.query('SELECT clamp(NaN, null, 1) as v')
+expected = [["_rowName","v"],["result","NaN"]]
+assert result == expected
+
+result = mldb.query('SELECT clamp(-NaN, null, 1) as v')
+expected = [["_rowName","v"],["result","-NaN"]]
+assert result == expected
+
+result = mldb.query('SELECT clamp(inf, null, 1) as v')
+expected = [["_rowName","v"],["result",1]]
+assert result == expected
+
+result = mldb.query('SELECT clamp(-inf, null, 1) as v')
+expected = [["_rowName","v"],["result","-Inf"]]
+assert result == expected
+
+result = mldb.query('SELECT clamp(null, null, 1) as v')
+expected = [["_rowName","v"],["result",None]]
+assert result == expected
+
+result = mldb.query('SELECT clamp(-2, -1, null) as v')
+expected = [["_rowName","v"],["result",-1]]
+assert result == expected
+
+result = mldb.query('SELECT clamp(0, -1, null) as v')
+expected = [["_rowName","v"],["result",0]]
+assert result == expected
+
+result = mldb.query('SELECT clamp(2, -1, null) as v')
+expected = [["_rowName","v"],["result",2]]
+assert result == expected
+
+result = mldb.query('SELECT clamp(nan, -1, null) as v')
+expected = [["_rowName","v"],["result",-1]]
+assert result == expected
+
+result = mldb.query('SELECT clamp(-nan, -1, null) as v')
+expected = [["_rowName","v"],["result",-1]]
+assert result == expected
+
+result = mldb.query('SELECT clamp(inf, -1, null) as v')
+expected = [["_rowName","v"],["result","Inf"]]
+assert result == expected
+
+result = mldb.query('SELECT clamp(-inf, -1, null) as v')
+expected = [["_rowName","v"],["result",-1]]
+assert result == expected
+
+result = mldb.query('SELECT clamp(null, -1, null) as v')
+expected = [["_rowName","v"],["result",None]]
+assert result == expected
+
+result = mldb.query('SELECT clamp(2, null, null) as v')
+expected = [["_rowName","v"],["result",2]]
+assert result == expected
+
+result = mldb.query('SELECT clamp(null, null, null) as v')
+expected = [["_rowName","v"],["result",None]]
+assert result == expected
+
+result = mldb.query('SELECT clamp(nan, null, null) as v')
```

Comment:
I would test valid cases where lower and higher are strings.  I would test failure cases where lower and higher are rows.